index.js

```diff
@@ -197,22 +197,16 @@ class ServerlessPlugin {
 
         const config = this.serverless.service;
         const isArmGlobally = config.provider.architecture === 'arm64';
-
-        // Check provider config
-        if (this.runtimes.includes(config.provider.runtime || '')) {
-            config.provider.layers = includeBrefLayers(
-                config.provider.runtime,
-                config.provider.layers || [], // make sure it's an array
-                isArmGlobally,
-            );
-            config.provider.runtime = 'provided.al2';
```

Comment:
By deleting this, we lose the global `provided.al2` which might be considered a BC break